jujugui/static/gui/src/app/components/scale-service/_scale-service.scss

```diff
@@ -0,0 +1,51 @@
+.scale-service {
+
+    input[type=text] {
```

Comment:
inputs are missing the inset box-shadow: http://ubuntudesign.github.io/vanilla-framework/demo/#forms. This should probably be set system wide.